src/client/nativeCommunication.ts

```diff
@@ -49,6 +51,22 @@ function getRect(slotPosition: DOMRect): Rect {
 	});
 }
 
+const getFormat: () => Option<Format> = () => {
```

Comment:
Why the `nativeCommunication` module out of curiosity? Do you anticipate that we'll start sending this to the native layers at some point?

Reply:
I don't think we will need to send format at some point since we're the ones handling all rendering .. the reason I needed this is to distinguish media articles so I can call the corresponding methods. But if we're ok to call the same methods for media articles (as per the comments above), we don't need any of this really.